pkg/xds/experimental/client/gcp/doc.go

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+// Package gcp provides helpers and cell definition specific to Google Cloud Project.
```

Comment:
Only relevant if we need to keep this package:
```suggestion
// Package gcp provides helpers and cell definition specific to Google Cloud Platform.
```